L/libavtp/build_tarballs.jl

```diff
@@ -0,0 +1,38 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libavtp"
+version = v"0.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/Avnu/libavtp/archive/refs/tags/v$(version).tar.gz", "9c1587431281a8d2404060e4bfd39f5cf0f211be6251e6afb1100e6148c1e591")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/libavtp-*
+mkdir build
+cd build
+meson --cross-file=${MESON_TARGET_TOOLCHAIN}
```

Comment:
https://github.com/Avnu/libavtp/blob/43a1f61565313166254ab92a39b53a005ac99ee2/meson.build#L45-L49
```suggestion
meson --cross-file=${MESON_TARGET_TOOLCHAIN} -Dtests=disabled
```

Reply:
WARNING: Unknown options: "tests"?

Reply:
https://github.com/Avnu/libavtp/blob/43a1f61565313166254ab92a39b53a005ac99ee2/meson_options.txt#L3 and the link above :shrug:

Reply:
I think the option was added after the most recent release, https://github.com/Avnu/libavtp/pull/24

Reply:
Then make cmocka a build dependency....